packages/now-node/src/dev-server.ts

```diff
@@ -0,0 +1,54 @@
+import http from 'http';
+import path from 'path';
+import { createServerWithHelpers } from './helpers';
+
+function listen(
+  server: http.Server,
+  port: number,
+  host: string
+): Promise<void> {
+  return new Promise(resolve => {
+    server.listen(port, host, () => {
+      resolve();
+    });
+  });
+}
+
+async function main() {
+  const entrypoint = process.env.NOW_DEV_ENTRYPOINT;
+  if (!entrypoint) {
+    throw new Error('`NOW_DEV_ENTRYPOINT` must be defined');
+  }
+
+  //const shouldAddHelpers = true;
+
+  const entrypointPath = path.join(process.cwd(), entrypoint);
+  const handler = await import(entrypointPath);
+
+  /*
```

Comment:
Is the commented code needed?

Reply:
Ya, it's still TODO (see edited OP).

Reply:
We're going to address this in a follow-up PR.